suds/client.py

```diff
@@ -729,7 +728,7 @@ def failed(self, binding, error):
 
     def location(self):
         p = Unskin(self.options)
-        return p.get('location', self.method.location)
+        return p.get('location', self.method.location.decode('utf-8'))
```

Comment:
Is it better to leave this as `.decode()`?

Reply:
it is the same as utf-8 is the default encoding as stated in [python3 docs](https://docs.python.org/3/library/stdtypes.html#bytes.decode)

Reply:
Is there a benefit to duplicating that?

Reply:
no